src/LazySets.jl

```diff
@@ -90,24 +90,25 @@ include("Rectification.jl")
 # =============================
 include("convert.jl")
 
+# =====================
+# Approximations module
+# =====================
+include("Approximations/Approximations.jl")
```

Comment:
This change should not be necessary. There should not be any dependency on the `Approximations` module.

Reply:
We added `using LazySets.Approximations: projection` in `is_empty_intersection.jl` and therefore it requires module`Approximation` to build LazySets.

Reply:
I see. I created #1429 to fix this later.